data/data/bootstrap/files/usr/local/bin/bootstrap-pivot.sh.template

```diff
@@ -41,26 +42,28 @@ if [ ! -f /opt/openshift/.pivot-done ]; then
 
   record_service_stage_start "rebase-to-okd-os-image"
 {{if .IsFCOS -}}
+  # get more space because 7.8gb is not enough
```

Comment:
Is this due https://github.com/openshift-metal3/dev-scripts/pull/1542? Sometimes ago we were affected by https://issues.redhat.com/browse/OCPBUGS-8036, but then it was fixed

Reply:
The increased space is required because container images could not be pulled due to insufficient storage and because i copy (rsync) all files from the machine os image to a podman volume (which had to do because the podman image would be "unmounted" at one point, quasi-erasing /usr)